src/geode/geosciences_io/model/helpers/geos_export.cpp

```diff
@@ -229,16 +229,16 @@ namespace geode
             auto box_node = root.append_child( "Box" );
             box_node.append_attribute( "name" ).set_value(
                 absl::StrCat( "well_", well_id++ ).c_str() );
-            box_node.append_attribute( "xMin" ).set_value(
-                absl::StrCat( "{", perf_box.min().value( 0 ) - global_epsilon,
-                    ", ", perf_box.min().value( 1 ) - global_epsilon, ", ",
-                    perf_box.min().value( 2 ) - global_epsilon, "}" )
-                    .c_str() );
-            box_node.append_attribute( "xMax" ).set_value(
-                absl::StrCat( "{", perf_box.max().value( 0 ) + global_epsilon,
-                    ", ", perf_box.max().value( 1 ) + global_epsilon, ", ",
-                    perf_box.max().value( 2 ) + global_epsilon, "}" )
-                    .c_str() );
+            box_node.append_attribute( "xMin" ).set_value( absl::StrCat( "{",
+                perf_box.min().value( 0 ) - 100. * global_epsilon, ", ",
```

Comment:
Never hardcode a value without documenting it (meaning with a named variable)

`static constexpr auto SAFETY_OFFSET = 100;` for example

Reply:
these variables are supposed to be defined at the beginning of the file